lib/jobs/generate-bmc-password.js

```diff
@@ -37,9 +37,13 @@ function bmcPasswordJobFactory(BaseJob, Logger, util, uuid, Promise,crypto) {
      */
     BmcPasswordJob.prototype._run = function _run() {
         var self = this;
-
+        //token is randomly generated password for the bmc credentials
         var token = crypto.randomBytes(4).toString('hex');
-        this.context.password = this.context.password || token;
+
+        //If the user doesn't enter a username and password, the default will be
+        //monorail for a user name and a random password(token)
+        this.context.password = this.options.password || this.context.password || token  ;
+        this.context.user = this.options.user || 'monorail' ;
```

Comment:
My opinion it's better to change the default user name to `rackhd`, or even better to a strange one like `__rackhd__` to avoid the conflict with user's existing username.